components/scream/src/physics/shoc/CMakeLists.txt

```diff
@@ -90,10 +90,22 @@ if (NOT SCREAM_DEBUG)
   endif()
 endif()
 
+if (HIP_BUILD)
+  #this is needed for crusher even with small kernels
+  set_source_files_properties(shoc_diag_second_shoc_moments_disp.cpp  PROPERTIES COMPILE_FLAGS -O1)
+endif()
+
 set(SHOC_LIBS "shoc")
 if (SCREAM_SMALL_KERNELS)
   add_library(shoc ${SHOC_SRCS} ${SHOC_SK_SRCS})
 else()
+
+  if (HIP_BUILD)
```

Comment:
Let's remove this. Below I pointed to the other part of the workaround. I want the workaround to be removed completely. Only small kernels works on HIP for now. Eventually, monolithic may work, too, but if it does, I want it to be without any compiler workarounds.

Reply:
thanks, done